modules/apps/notification/notification-api/src/main/java/com/liferay/notification/constants/NotificationRecipientSettingConstants.java

```diff
@@ -32,6 +32,8 @@ public class NotificationRecipientSettingConstants {
 
 	public static final String NAME_TO_TYPE = "toType";
 
+	public static final String NAME_USE_USER_LOCALE = "useUserLocale";
```

Comment:
@pedro-oliveira446 rename to `useMostRelevantLocaleForGuestUsers`, I chose to follow the same pattern used in `maximumFileSizeForGuestUsers`

Reply:
Done!